Upgrade.md

```diff
@@ -0,0 +1,86 @@
+# Upgrade
+
+All instructions to  upgrade this project from  one release to the  next will be
+documented in this  file. Upgrades must be run sequentially,  meaning you should
+not skip minor/major releases while upgrading (fix releases can be skipped).
+
+The format is inspired from [Keep a
+Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
+[Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## Unreleased
+
+### Static services
+
+This version introduces `Static services` and it is a breaking change in the
+blue/green deployment process. The `switch` playbook is patching static services
+instead of routes. And the `deployment_stamp` of the deployed stacks is no
+longer stored in routes, but in static services instead.
+
+In order to migrate to this new system, you have to apply the following upgrade
+process. This will not cause any downtime, but it implies a redeployment of your
+applications.
+
+#### Upgrade Procedure
+
+This procedure must be applied for every arnold application deployed.
+
+The `current` stack of your application (before applying this procedure) will be
+referenced as `BEFORE-UPGRADE-VERSION`.
+
+
+##### 1. Upgrade your arnold version
+
+##### 2. Execute the `create_services` ansible playbook :
+
+```
+bin/ansible-playbook create_services.yml -e "apps_filter=your-application"
+```
+It will create the static  services required by your application. Each variant of these static services (`previous`, `current`, `next`) willtarget nothing after this step.
```

Comment:
```suggestion
It will create the static services required by your application. Each variant of these static services (`previous`, `current`, `next`) will target nothing after this step.
```